TicTacToe/Program.cs

```diff
@@ -1,31 +1,31 @@
 ﻿using System;
 
+
+// Do the same thing with this snippet from TicTacToe/Program.cs:
 namespace TicTacToe
 {
     class Program
-    {
-
-
-
-        
-
-        static int dimension = 3;
-        private static char emptyCell = ' ';
+    { 
+        // the dimenstion is the most important aspect of this game
+        static int dimension = 3; 
```

Comment:
shouldn't this dimension be 2?

Reply:
I think it should be changed to 2 actually.